.github/workflows/ci.yaml

```diff
@@ -205,13 +212,30 @@ jobs:
       - name: Setup
         uses: ./.github/workflows/composite/setup
 
+      - name: Checkout FRR-K8s
+        uses: actions/checkout@v4
+        if: matrix.bgp-type == 'frr-k8s-external'
+        with:
+          repository: metallb/frr-k8s
+          path: frr-k8s
+          ref: v0.0.11
+
+      - name: Deploy FRR-K8s
+        if: matrix.bgp-type == 'frr-k8s-external'
+        run: |
+          cd ${GITHUB_WORKSPACE}/frr-k8s
+          make docker-build
+          make deploy
```

Comment:
can we deploy the all-in-one for that version instead of checkout+build+.. ?

Reply:
ah yes, that's much better. I stuck to what I had in my local dev env but this is way more straightforward!

Reply:
I guess I can do everything from dev-env then